network-api/networkapi/buyersguide/management/commands/generate_pni_report.py

```diff
@@ -0,0 +1,84 @@
+import psycopg2
+from urllib.parse import urlparse
+from django.core.management.base import BaseCommand
+from django.conf import settings
+
+from networkapi.buyersguide.models import Product
+
+
+class Command(BaseCommand):
+    help = 'Generate statistical data about privacynotincluded activity, for internal' \
+           ' dashboards and update a DB connected to data studio'
+
+    def setup_db_connection(self):
+        STATS_DB = urlparse(settings.PNI_STATS_DB_URL)
+        return psycopg2.connect(
```

Comment:
Missing a short and sweet `if STATS_DB is '': return print('missing PNI_STATS_DB_URL environment variable')` or the like.